frontend/src/workStation/WorkStation.jsx

```diff
@@ -58,7 +59,7 @@ function WorkStation() {
       <Box>
         <CodeIDE fileContent={fileData.fileContent} />
       </Box>
-      <div>This is group: {groupData.groupName}</div>
+      <div>This is group: {groupId}</div>
```

Comment:
It looks like this commit is reverting updates you made.

Reply:
It must have happened during the merge conflict resolution. I made a another commit to fix this: https://github.com/izuchukwumba/peered/pull/11/commits/bf6f046f1ea02885b098dbba39c3323a007d85cb